checker/src/main/java/org/checkerframework/checker/signedness/SignednessVisitor.java

```diff
@@ -154,16 +154,18 @@ public Void visitBinary(BinaryTree node, Void p) {
                     AnnotationMirror rightAnno =
                             rightOpType.getEffectiveAnnotations().iterator().next();
 
-                    if (leftOpType.getKind() != TypeKind.CHAR
-                            && !TypesUtils.isDeclaredOfName(
-                                    leftOpType.getUnderlyingType(), "java.lang.Character")
+                    // Note that leftOpType.getUnderlyingType() and rightOpType.getUnderlyingType()
+                    // are always java.lang.String. Please refer to binaryTreeArgTypes for more
+                    // details.
```

Comment:
A comment here would be useful to highlight that `xOriginalType` in contrast can be something other than string. At the moment it's not clear to me why we need these extra variables otherwise.

Reply:
Done